ogre2/src/Ogre2Scene.cc

```diff
@@ -48,6 +48,9 @@ class ignition::rendering::Ogre2ScenePrivate
 {
   /// \brief Flag to indicate if shadows need to be updated
   public: bool shadowsDirty = true;
+
+  /// \brief A list of particle emitters
+  public: std::vector<ParticleEmitterPtr> particleEmitters;
```

Comment:
Do we need this? It looks like the only other time it's used is in `Ogre2Scene::CreateParticleEmitterImpl`: https://github.com/ignitionrobotics/ign-rendering/blob/b73e9d6d1c7d123775680377fced95a02737ec76/ogre2/src/Ogre2Scene.cc#L460

I think we might be able to remove this data member.

Reply:
not used so removed. f7b1838